PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -139,6 +141,7 @@ struct HfTrackIndexSkimCreatorTagSelCollisions {
       labels[0] = "processed";
       labels[1] = "selected";
       labels[2 + EventRejection::Trigger] = "rej. trigger";
+      labels[2 + EventRejection::TimeFrameBorderCut] = "time-frame border";
```

Comment:
```suggestion
      labels[2 + EventRejection::TimeFrameBorderCut] = "rej. time-frame border";
```
@fgrosa Minor, I let you decide if you want to merge now or implement the change and restart the tests